Unix/installbuilder/service_scripts/service_control.linux

```diff
@@ -127,6 +138,12 @@ restart_omi()
         return
     fi
 
+    if [ "$OMI_DISABLE_SERVICE_START" ]; then
+        stop_omi
+        start_omi
```

Comment:
I am not sure if we need to wait omi stopped, or if it is possible a timing issue here, just pick it up.